ot-node.js

```diff
@@ -71,7 +72,7 @@ class OTNode {
     }
 
     initializeLogger() {
-        this.logger = new Logger(this.config.logLevel, this.config.telemetry.enabled);
```

Comment:
Why did we removed this part?

Reply:
the telemetry enabled flag is not needed in the logger constructor anymore